staging/2021/benchmarks/source.csv

```diff
@@ -1,5 +1,4 @@
 Measure Title,Measure ID,Collection Type,Measure Type,High Priority,Average Performance Rate,Measure has a Benchmark,Decile 3,Decile 4,Decile 5,Decile 6,Decile 7,Decile 8,Decile 9,Decile 10,Topped Out,Seven Point Cap
-Diabetes: Hemoglobin A1c (HbA1c) Poor Control (>9%),1,Medicare Part B Claims,Intermediate Outcome,Y,15.6727392550143,Y,79.99 - 70.01,70 - 60.01,60 - 50.01,50 - 40.01,40 - 30.01,30 - 20.01,20 - 10.01,<= 10,N,N

```

Comment:
Same deal here and on line 60. You'll keep the rows, but clear out the decile info and mark `Measure has a Benchmark=N`